Library/Homebrew/utils/analytics.rb

```diff
@@ -1,34 +1,52 @@
-
-def analytics_anonymous_prefix_and_os
+def analytics_label
   @analytics_anonymous_prefix_and_os ||= begin
-    "#{OS_VERSION}, #{HOMEBREW_PREFIX.to_s.gsub(ENV["HOME"], "~")}"
+    os = OS_VERSION
+    prefix = ", non-/usr/local" if HOMEBREW_PREFIX.to_s != "/usr/local"
```

Comment:
Have changed the screenshot. Would rather keep the prefix and CI consistent with each other, I think, and avoid printing "the expected case"